fhir-bench-orchestrator/src/test_framework/mod.rs

```diff
@@ -100,8 +100,15 @@ pub struct ServerOperationLog {
 }
 
 /// Represents the unique name of a FHIR server operation that this framework tests.
-#[derive(SerdeValue, Clone, Serialize)]
-pub struct ServerOperationName(pub &'static str);
+#[derive(Deserialize, SerdeValue, Clone, Serialize)]
+pub struct ServerOperationName(pub String);
+
+impl From<&str> for ServerOperationName {
```

Comment:
Should have a comment here or on the struct demonstrating how to create `ServerOperationName`s.